sphinx_gallery/gen_gallery.py

```diff
@@ -247,11 +248,12 @@ def generate_gallery_rst(app):
 
             fhindex.write(SPHX_GLR_SIG)
 
-    if gallery_conf['plot_gallery']:
+    if gallery_conf['plot_gallery'] and gallery_conf['min_reported_time']:
```

Comment:
I don't understand why you are adding `and gallery_conf['min_reported_time']`. The vaiue of min_reported_time is 0 so this does not log by default, right?

Reply:
Good catch, probably leftover from earlier design.